.github/workflows/enclave-cc-cicd.yaml

```diff
@@ -0,0 +1,59 @@
+name: CI/CD | enclave-cc e2e tests
+on:
+  schedule:
+  - cron: "0 0 * * *"
+
+permissions:
+  contents: read
+
+jobs:
+  e2e:
+    name: operator enclave-cc e2e tests
+    runs-on: ubuntu-22.04
+    services:
+      registry:
+        image: registry:2
+        ports:
+          - 5000:5000
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+        with:
+          driver-opts: network=host
+
+      - name: Build and push to local registry
+        uses: docker/build-push-action@v3
+        with:
+          context: .
+          push: true
+          tags: localhost:5000/cc-operator:latest
+
+      - name: Setup kind cluster
+        run: |
+          kind create cluster -n coco-sgx --config tests/e2e/enclave-cc-kind-config.yaml
+          kubectl label node coco-sgx-worker node-role.kubernetes.io/worker=
+
+      - name: Deploy operator from the local registry
+        run: |
+          cd config/default
+          kustomize edit set image localhost:5000/cc-operator:latest
+          kubectl apply -k .
+          sleep 1
+          kubectl wait --for=jsonpath='{.status.conditions[0].status}'=True deployments/cc-operator-controller-manager -n confidential-containers-system
+
+      - name: Install enclave-cc sim
+        run: |
+          current_tag=$(curl -X GET "https://quay.io/api/v1/repository/confidential-containers/runtime-payload/tag/" | jq .tags[].name | grep enclave-cc-SIM | grep -v latest | head -1 | sed -e 's|\"||g')
+          latest_ci_tag=$(curl -X GET "https://quay.io/api/v1/repository/confidential-containers/runtime-payload-ci/tag/" | jq .tags[].name | grep enclave-cc-SIM | grep -v latest | head -1 | sed -e 's|\"||g') 
+          sed -e -i 's|runtime-payload:${current_tag}|runtime-payload-ci:${latest_ci_tag}/g' config/samples/ccruntime-enclave-cc-sim.yaml
+          echo "Testing Enclave CC using the following CI tag: ${latest_ci_tag}"
```

Comment:
@mythi, could I use `kustomize` to do such change?

Reply:
we're missing `kustomization`s for enclave-cc `CcRuntime` but I can see that could indeed help: it would be possible to set `payloadImage` easily to any (sim/hw/CI/published) images